ui/app/src/services/content.ts

```diff
@@ -1463,16 +1464,16 @@ export function getChildrenByPath(
   );
 }
 
-export function copy(site: string, item: Partial<LegacyItem>): Observable<{ success: boolean }> {
-  let _item = item.children ? { item: [item] } : { item: [{ uri: item.path }] };
+export function copy(site: string, path: string, copyItem?: CopyItem): Observable<{ success: boolean }> {
+  let item = path ? { item: [{ uri: path }] } : { item: [copyItem] };
```

Comment:
Looks like you’d be sending [undefined] when no copyitem is supplied

Reply:
yes, CopyItem is not needed when you already sent a path